code/modules/clothing/under/miscellaneous.dm

```diff
@@ -104,6 +104,16 @@
 	sensor_mode = SENSOR_COORDS
 	random_sensor = FALSE
 
+/obj/item/clothing/under/rank/deathsquad
+	name = "\improper Deathsquad jumpsuit"
+	desc = "It's a jumpsuit worn by the Deathsquad, providing slight protection. A small tag at the bottom reads \"Not associated with Nanotrasen\". "
+	icon_state = "officer"
+	item_state = "g_suit"
+	item_color = "officer"
+	armor = list(MELEE = 20, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 10, BIO = 0, RAD = 0, FIRE = 30, ACID = 30)
```

Comment:
this armour value will make Dsquad **IMMUNE** to melee attacks, outside of armour pen. 20 + 80 = 100.
while Dsquad is supposed to be mega powerful. I don't think making it so Tspiders, blobs, physically cannot damage them is a good idea.